### PR TITLE
goose 2.6.0

### PR DESCRIPTION
**Pull request by chenrui333**

### CODE REVIEW

**Comment by SMillerDev** [comment]
I think the variable should be about go11, not 111

**Comment by chenrui333** [comment]
@SMillerDev which variable did you refer to?

**Comment by SMillerDev** [comment]
Yes, think so

**Comment by chenrui333** [comment]
oh, that is just the common setup for almost `go modules` related formulas.

**Comment by SMillerDev** [comment]
I thought those were GO11?

**Comment by chenrui333** [comment]
I am still not quite clear about your suggestion.

But, from what I can see, `GO111MODULE` is very common build setup option?
https://github.com/search?q=%22GO111MODULE%22+org%3Ahomebrew&type=Code

**Comment by chenrui333** [comment]
yeah, I see, you are mostly referring to this?
https://github.com/pressly/goose/blob/master/.travis.yml#L4

**Comment by SMillerDev** [comment]
>I am still not quite clear about your suggestion. ...

@chenrui333 I guess I was wrong then. Carry on

**Comment by chenrui333** [comment]
This does fit into my golang expertise along with maintaining so many golang formula, going back working on this now.

**Comment by chenrui333** [comment]
Should be good now, cc @SMillerDev @lembacon @Moisan

**Comment by chenrui333** [comment]
Not sure why I made some unneeded changes earlier.

**Comment by chenrui333** [comment]
15:20:12 1 gem installed
15:20:12 Error: 1 problem in 1 formula detected
15:20:12 goose:
15:20:12   * C: 23: col 7: use "dep", "ensure", "-vendor-only"

**Comment by chenrui333** [comment]
00:54:47 ==> dep ensure -vendor-only
00:54:47 no Gopkg.lock exists from which to populate vendor/
00:54:47 
00:54:47 ==> Formula

File: Formula/goose.rb [hunk 1]
```diff
@@ -17,9 +17,10 @@ class Goose < Formula
 
   def install
     ENV["GOPATH"] = buildpath
+
     (buildpath/"src/github.com/pressly/goose").install buildpath.children
     cd "src/github.com/pressly/goose" do
-      system "dep", "ensure"
```
Comment (chenrui333) [inline Formula/goose.rb]: @SMillerDev How can I bypass this change?

Comment (chenrui333) [reply Formula/goose.rb]: relates to this PR, https://github.com/Homebrew/brew/pull/4305

**Comment by chenrui333** [comment]
I dont think upstream repo is ready-ready yet with `go mod` and current `dep ensure` does not with brew auditing system. 

Closing issue now.
relates to #34207, pressly/goose#110

**Pull request closed without merging**